runtime/compiler/CMakeLists.txt

```diff
@@ -358,6 +358,10 @@ create_omr_compiler_library(NAME j9jit
 if(OMR_OS_ZOS)
 	# Workaround a compile problem on z/OS by appending "-O" (to override "-O3").
 	set_property(SOURCE "optimizer/VectorAPIExpansion.cpp" APPEND PROPERTY COMPILE_FLAGS "-O")
+	if(CMAKE_C_COMPILER_IS_OPENXL)
```

Comment:
Why doesn't this use `OMR_TOOLCONFIG` (like in #19880)?
```
	if(OMR_TOOLCONFIG STREQUAL "openxl")
```

Reply:
Should be using `OMR_TOOLCONFIG`, I had forgot the one I was using across PRs post holidays. I made the recommended changes.

Reply:
I see line 287 was changed, but not this line.